core/src/main/java/com/sequenceiq/cloudbreak/reactor/handler/cluster/dr/BackupRestoreSaltConfigGenerator.java

```diff
@@ -17,17 +18,31 @@
 
 @Component
 public class BackupRestoreSaltConfigGenerator {
+    // these values are tightly coupled to the `postgresql/disaster_recovery.sls` salt pillar in `orchestrator-salt`
+    public static final String POSTGRESQL_DISASTER_RECOVERY_PILLAR_PATH = "/postgresql/disaster_recovery.sls";
```

Comment:
This diesn't need to be public, others visibility can be lowered also to package level.

Reply:
I'm merging the PR. We can address this later.